packages/ffe-file-upload/less/file-upload.less

```diff
@@ -0,0 +1,307 @@
+        padding-right: 20px;
+        padding-top: 5px;
+        margin-left: auto;
+        background-color: transparent;
+        border: none;
+        line-height: 18px;
+        position: relative;
+        color: @ffe-blue-azure;
+        font-size: 14px;
+        grid-column: 3;
+        grid-row: 1;
+
+        @media (min-width: @breakpoint-md) {
+            margin-right: 20px;
+        }
+
+        &:active,
+        &:focus,
+        &:hover {
+            text-decoration: underline;
+            color: @ffe-blue-royal;
+
+            > .ffe-file-upload__file-item-delete-icon {
+                background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='14' height='14' viewBox='0 0 200 200'%3E%3Cpath fill='@{ffe-blue-royal-fill}' d='M18.77,0c-3.562,0-7.125,1.347-9.834,4.044l-4.429,4.439c-5.381,5.394-5.381,14.25,0,19.68l71.19,71.83-71.66,71.83c-5.381,5.394-5.381,14.25,0,19.68l4.429,4.439c5.381,5.394,14.22,5.394,19.64,0l71.66-71.83,71.65,71.83c5.381,5.394,14.23,5.394,19.65,0l4.429-4.439c5.381-5.394,5.381-14.25,0-19.68l-71.19-71.83,71.66-71.83c5.381-4.914,5.381-13.78,0-19.21l-4.429-4.423c-5.381-5.394-14.22-5.394-19.64,0l-72.13,71.34-71.17-71.83c-2.691-2.697-6.257-4.044-9.819-4.044z'/%3E%3C/svg%3E");
+            }
+
+            ~ .ffe-file-upload__file-item-filename {
+                text-decoration: underline;
+            }
+
+            ~ .ffe-file-upload__file-item-error-info
+                > .ffe-file-upload__file-item-error-info-filename {
+                text-decoration: underline;
+            }
+
+            ~ .ffe-file-upload__file-item-stencil-info
+                > .ffe-file-upload__file-item-stencil-info-filename {
+                text-decoration: underline;
+            }
+        }
+    }
+
+    &__file-item-stencil {
+        display: grid;
+        grid-template-columns: 1fr;
+
+        &-info {
+            display: grid;
+            grid-column: 1;
+            grid-row: 1;
+            grid-template-columns: 1fr;
+            overflow: hidden;
+
+            &-background {
+                height: 10px;
+                border-radius: 5px;
+                background-color: #ebebeb;
+            }
+
+            &-filename {
+                &:extend(.ffe-micro-text);
+
+                margin-top: 5px;
+                white-space: nowrap;
+                overflow: hidden;
+                text-overflow: ellipsis;
+            }
+        }
+    }
+
+    &__file-item-loaded {
+        display: grid;
+        grid-template-columns: 1fr;
+    }
+
+    &__file-item-filename {
+        grid-row: 1;
+        grid-column: 1;
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        margin-top: 2px;
+        color: @ffe-blue-royal;
+    }
+
+    &__file-item-error {
+        display: grid;
+        grid-gap: 5px;
+        grid-template-columns: 25px 1fr;
+
+        &-icon {
+            &:extend(.ffe-field-error-message);
+
```

Comment:
The definition of `.ffe-field-error-message` is provided by `ffe-form`, but `ffe-form` is not listed as dependency in `package.json`.

Reply:
Nice catch!